src/celeritas/inp/Physics.hh

```diff
@@ -72,6 +73,9 @@ struct OpticalPhysics
     //! Generate scintillation photons
     bool scintillation{true};
     //!@}
+
+    //! Volume labels for sensitive detectors
+    std::optional<std::vector<Label>> detector_labels;
```

Comment:
Can we pass `SDParams` through from the core params? This isn't exactly physics input.

Reply:
Yeah, definitely not the right place. I could put it in `inp::Scoring`?